tensorflow/core/kernels/fused_batch_norm_op.cu.cc

```diff
@@ -358,7 +375,11 @@ struct FusedBatchNormInferenceFunctor<GPUDevice, T, U> {
     if (count == 0) return;
 
     bool launched = false;
+    #if TENSORFLOW_USE_ROCM
+    constexpr int32 kThreadInBlock = 1024;
```

Comment:
wondering why the CUDA side is set to 512 in the first place!

Reply:
Maybe 512 is optimal for their arch. Here it works with 512 but seems to be a bit faster with 1024. (Although GCN docs state that 1024 is a suboptimal value for occupancy purposes, so this may warrant further benchmarking.)